lib/http/2/huffman_statemachine.rb

```diff
@@ -10,263 +10,278 @@ class Huffman
       # :nodoc:
       MAX_FINAL_STATE = 7
       MACHINE = [
-        [[nil,16],[nil,76],[nil,54],[nil,36],[nil,30],[nil,28],[nil,24],[nil,221],[nil,9],[nil,10],[nil,11],[nil,12],[nil,13],[nil,14],[nil,15],[nil,1],],
```

Comment:
I managed to tweak the generation script to emit rubocop-accepted code, what's the harm?